notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -695,7 +734,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.12"
+   "version": "3.13.2"
```

Comment:
Looks like [3.13 still isn't supported](https://github.com/pytorch/pytorch/issues/130249) for some things?

Reply:
thanks! fixed this